docs/jax-101/06-parallelism.md

```diff
@@ -0,0 +1,423 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.0
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
++++ {"id": "tCOWitsAS1EE"}
+
+# Parallel evaluation in JAX
+
+[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/jax/blob/master/docs/jax-101/06-parallelism.ipynb)
+
+*Authors: TODO*
+
+In this section we will discuss the facilities built into JAX for single-program, multiple-data (SPMD) code.
```

Comment:
Dumb question (sorry) @jakevdp: In https://github.com/google/jax/pull/5958/files, you added SPMD to the JAX glossary and defined it as "single program multi data", whereas here it's "single-program, multiple-data". Is there more than one way of writing SPMD?